packages/agreement-process/package.json

```diff
@@ -31,7 +31,7 @@
     "pagopa-interop-commons-test": "workspace:*",
     "pg-promise": "11.8.0",
     "prettier": "2.8.8",
-    "puppeteer": "22.10.0",
+    "puppeteer": "22.11.0",
```

Comment:
Unrelated, but also fixing the fact that we were using different versions of puppeteer in different packages